.github/workflows/wool.yml

```
@@ -0,0 +1,17 @@
+on:
+  pull_request:
+  issue_comment:
+    types: [created, edited]
```

```suggestion
  pull_request
```
the issue_comment thing is broken